.github/workflows/deploy.yml

```
@@ -136,7 +136,7 @@ jobs:
 
       - name: Setup | Release notes
         run: |
-          GO111MODULE=on go get github.com/git-chglog/git-chglog/cmd/git-chglog@0.9.1
+          GO111MODULE=on go get github.com/git-chglog/git-chglog/cmd/git-chglog@0.15.0
```

If you're updating the chglog version, you might also need to update the go version.

---

In addition, the tag needs to be changed and it looks like `go get` has been deprecated in favour of `go install`:
```suggestion
          go install github.com/git-chglog/git-chglog/cmd/git-chglog@v0.15.0
```

---

I've bumped the version of go to the same version that `git-chglog` use in their [CI workflow](https://github.com/git-chglog/git-chglog/blob/master/.github/workflows/ci.yml#L15).

Thanks I didn't know `go get` was deprecated, I've been out of the Go ecosystem for longer than I thought.

---

I think the warning was added with go 1.17, so this seems to a fairly recent development.